### PR TITLE
Added basic calculator script in .js and site in .html

### PR DESCRIPTION
**Pull request by jkn1k**

### CODE REVIEW

File: kalkulator/script.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+document.getElementById('calculator').addEventListener('submit', function(event) {
+
+    event.preventDefault();
+
+    let liczba1 = parseFloat(document.getElementById('liczba1').value);
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/script.js]: I would use const here

File: kalkulator/script.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    let liczba2 = parseFloat(document.getElementById('liczba2').value);
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/script.js]: I would use const here

File: kalkulator/script.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    let operacja = document.getElementById('operacja').value;
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/script.js]: I would use const here

File: kalkulator/script.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+    let wynik;
+
+    if (operacja === 'dodawanie') {
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/script.js]: expected loop here will be switch case

File: kalkulator/script.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        wynik = liczba1 + liczba2;
+    } else if (operacja === 'odejmowanie') {
+        wynik = liczba1 - liczba2;
+    } else if (operacja === 'mnozenie') {
+        wynik = liczba1 * liczba2;
+    } else if (operacja === 'dzielenie') {
+        wynik = liczba1 / liczba2;
+    }
+
+    document.getElementById('wynik').textContent = `Wynik: ${wynik}`;
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/script.js]: we also expect the result to be a comment so as not to divide by zero

Comment (kamilzawadzki-tomtom) [reply kalkulator/script.js]: Using "textContent" will return the complete text content, including the hidden text. However, using "innerText" will only return the visible text content, excluding the hidden text.

File: kalkulator/index.html [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+    }
+    button:hover {
+        background-color: #223e97;
+    }
+</style>
+</head>
+<body>
+    <h1>Kalkulator</h1>
+    <form id="calculator">
+        <label for="liczba1">Pierwsza liczba:</label>
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/index.html:51]: u can use .placeholder,  then you have 1 line of code
https://www.w3schools.com/jsref/prop_text_placeholder.asp

File: kalkulator/index.html [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+        <input type="number" id="liczba1" required><br>
+
+        <label for="liczba2">Druga liczba:</label>
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/index.html:54]: the same here

File: kalkulator/index.html [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+            <option value="dodawanie">Dodawanie</option>
+            <option value="odejmowanie">Odejmowanie</option>
+            <option value="mnozenie">Mnożenie</option>
+            <option value="dzielenie">Dzielenie</option>
+        </select><br>
+
+        <button type="submit">Oblicz</button>
+    </form>
+
+    <h2 id="wynik"></h2>
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/index.html]: this line can be like this? please check :)
`<h2>Wynik: <span id="result"></span></h2>`

File: kalkulator/script.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+    let wynik;
+
+    if (operacja === 'dodawanie') {
+        wynik = liczba1 + liczba2;
+    } else if (operacja === 'odejmowanie') {
+        wynik = liczba1 - liczba2;
+    } else if (operacja === 'mnozenie') {
+        wynik = liczba1 * liczba2;
+    } else if (operacja === 'dzielenie') {
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/script.js]: use switch case and 
when u divide you can also add a comment: 'Nie można dzielić przez zero!

**Review by kamilzawadzki-tomtom — CHANGES_REQUESTED** [review]

File: kalkulator/script.js [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+            break;
+        case 'mnozenie':
+            wynik = liczba1 * liczba2;
+            break;
+        case 'dzielenie':
+            if (liczba2 !== 0) {
+                wynik = liczba1 / liczba2;
+            } else {
+                wynik = 'Nie można dzielić przez zero!';
+            }
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/script.js:26]: pls add default

**Review by kamilzawadzki-tomtom — CHANGES_REQUESTED** [review]

File: kalkulator/script.js [hunk 11]
```diff
@@ -0,0 +1,33 @@
+document.getElementById('calculator').addEventListener('submit', function(event) {
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/script.js:1]: this file is missing the following: break and line endings. before sending the code please make sure it works

File: kalkulator/script.js [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+                wynik = liczba1 / liczba2;
+            } else {
+                wynik = 'Nie można dzielić przez zero!';
+            }
+            default:
+                wynik = "Nieprawidłowa operacja"
+    }
+    
+
+    document.getElementById('wynik').innerText = `${wynik}`;
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/script.js]: you can just use: wynik. check if it works.

File: kalkulator/index.html [hunk 13]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
```
Comment (kamilzawadzki-tomtom) [inline kalkulator/index.html:1]: please close comments as resolved or add comments because it is not known whether you have fixed it or not.

**Review by kamilzawadzki-tomtom — CHANGES_REQUESTED** [review]

File: kalkulator/script.js (hunk 11 above)
Comment (kamilzawadzki-tomtom) [reply kalkulator/script.js:1]: at the moment the .js script does not work correctly for the operation: divide.

File: kalkulator/script.js (hunk 12 above)
Comment (jkn1k) [reply kalkulator/script.js]: Works.

**Review by kamilzawadzki-tomtom — APPROVED** [review]
LGTM

**Pull request merged**